### ISSUE TITLE
[amateur analysis] OpenVR chooses incorrect Primary display for (some?) overlays when using a Windows Mixed Reality HMD without having a display connected to the GPU.

### ISSUE THREAD

**Issue by TheJokingTeen** [ValveSoftware/openvr#1634]
This is my theory, I've come to this assumption because disabling virtual display device in device manager makes the Steam overlay appear again, at the cost of insane stuttering.

Some overlays still work without this workaround, like accessing game info by opening its details inside SteamVR Home.

I'm having issues logging this as I can't create a system report without access to my monitor...

**Comment by okawo80085** [comment]
Ok, you need to clarify situation, what exactly is broken? Is you Status window now showing up? Is you System overlay not working? 
If you're not sure, attach some screenshots or record a short video, additional context is really needed here.

**Comment by TheJokingTeen** [comment]
> Ok, you need to clarify your situation, what exactly is broken? Is you Status window not showing up? Is you System overlay not working? If you're not sure, attach some screenshots or record a short video, additional context is really needed here.

Right, so, the SteamVR overlay does not load in when triggered, but it still gets registered, as the view still gets dim and blurred, but the whole overlay window containing all the buttons to access games, settings, plugins etc is completely invisible. Pointing the controller towards the target position of the overlay's elements reveals the pointer, suggesting that the points to render the overlay were processed, but it didn't render in.

This might have been caused by my buggy workaround of an issue for using wmr at 90hz (because it decides I'm not allowed to :|) so I'll test this at 60hz to make sure this isn't just me. 

I hope that's a little clearer, but I'll try to get logs or a recording. Anyone know how to create system reports automatically instead of doing it manually?

**Comment by okawo80085** [comment]
This clarifies your situation *a lot*, still need at least some logs though.

>  but I'll try to get logs or a recording.

SteamVR dumps logs to the general Steam log folder, so if you're on windows it's most likely in `Program Files x86\steam\logs`,  look for anything starting with "vr", but most of the time you are either interested in `vrserver.txt`, `vrcompositor.txt` or `vrdashboard.txt`, there is also a web console which you can get to through your Status Window -> Developer -> Web Console.

> Anyone know how to create system reports automatically instead of doing it manually?

Afaik that's not possible.
